wrapping/python/openmeeg/tests/test_mesh.py

```diff
@@ -1,11 +1,11 @@
 import os
 import numpy as np
-import openmeeg._openmeeg_cxx as _omc
+from openmeeg.openmeeg import _Mesh as Mesh
```

Comment:
Yes you could prefix with `_` to make it `openmeeg._openmeeg`, but once you do that there is no reason to do any renaming of the wrapped C++ items to `openmeeg._openmeeg._Vect3` instead of `openmeeg._openmeeg.Vect3`, as the `Vect3` class is now private *unless we import it into some public namespace* (which we shouldn't). And at that point you're pretty much back to where we are on `main`, just with `openmeeg._openmeeg` instead of `openmeeg._openmeeg_cxx`. So I don't see much point.